MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -63,6 +63,7 @@ Use these conventions to indicate the status of each criterion.
   * org.z3950.zing:cql-java is allowed if appropriately labelled, even if it is LGPL-2.1-only
   * org.marc4j:marc4j is allowed if appropriately labelled, even if it is LGPL-2.1-or-later
   * org.hibernate.* is allowed if appropriately labelled, even if it is LGPL-2.1-or-later
+  * _note: If a library declares multiple licenses in its pom.xml, [only one of them needs to comply](https://maven.apache.org/ref/3.9.11/maven-model/maven.html#project)._
```

Comment:
I agree this is a useful clarification to add to the criteria

> If a library declares multiple licenses in its pom.xml, [only one of them needs to comply]

I also like the effort to be specific about where those licenses are declared, to help evaluators.  But there was concern expressed in the meeting today about the reference to the pom.xml being java-specific, which excludes non-java server modules as well as all client modules (this is in Shared/Common).  

Once the new subgroup documents (hopefully) some automated tooling for the various scenarios, the evaluator would hopefully not have to check pom.xml manually.  So I wonder if for now we should just have the general clarification, and leave off the pom.xml detail and link?